### DISCUSSION TITLE
is this project open to Hacktoberfest submissions?

### DISCUSSION THREAD

**Discussion by Varix** [ggirelli/fastx-barber#125] [General]
I have a bunch of Bioinformatics students looking to get involved, I remember in the past people were trying stuff here but with the new rules of projects having to opt in I am not sure if fastx-barber was flagged for people to help out

**Comment by ggirelli** [comment]
Hi @Varix, thank you for your question. Yes, this repository is indeed open for hacktoberfest submissions! Feel free to look for/discuss in any issues that you would like to address, or to submit a PR with a new feature you would like to see included in the package :)
